charts/kubernetes-agent/templates/nfs-pv.yaml

```diff
@@ -15,6 +15,9 @@ spec:
   mountOptions:
     - nfsvers=4.1
     - lookupcache=none
+    - soft
+    - timeo=5
```

Comment:
this is 500ms?

Reply:
yep - after speaking with @liam-mackie I'm going to change these values to `timeo=100` and `retrans=6` so it's approximately 1 minute of retries before a pod will fail.